apps/participants/views.py

```diff
@@ -137,6 +138,18 @@ def participant_team_detail(request, pk):
     elif request.method in ["PUT", "PATCH"]:
 
         if request.method == "PATCH":
+            if not is_user_creator_of_participant_team(
+                request.user, participant_team
+            ):
+                response_data = {
+                    "error": [
```

Comment:
Value of `error` key should be string instead of a list, no?

Reply:
On the frontend side we expect array of errors and show only error at 0th index. I think it is because we only used to return serializer errors from this API

Reply:
okay, but shouldn't we make a change on the frontend checking if `error` is present then we are displaying the string?
Sending the value as a list doesn't look right to me.